.github/workflows/test-mingw.yml

```diff
@@ -80,7 +80,7 @@ jobs:
           pushd depends && ./install_extra_test_images.sh && popd
 
       - name: Build Pillow
-        run: SETUPTOOLS_USE_DISTUTILS="stdlib" CFLAGS="-coverage" python3 -m pip install --global-option="build_ext" .
+        run: SETUPTOOLS_USE_DISTUTILS="stdlib" CFLAGS="-coverage" python3 -m pip install .
```

Comment:
I got to the conclusion that it's also nice to configure coverage  through the config setting. Here's an example of the interface I ended up with: https://yarl.aio-libs.org/en/latest/changes/#released-versions

Reply:
We're already providing `make install-coverage`. I don't see the need to provide yet another way of enabling coverage.

Reply:
Yeah, I see. Just thought it's less integrated into the Python interfaces, but into tooling of external ecosystems.